index.ts

```diff
@@ -97,25 +97,31 @@ function toZodShape<Options extends z.ZodRawShape, Flags extends z.ZodRawShape,
 
 export function _parse<Options extends z.ZodRawShape, Flags extends z.ZodRawShape, Positional extends PositionalTuple>(schema: ZodCliSchema<Options, Flags, Positional>, args: string[]) {
   const obj = toObject(schema, args);
-  return toZodShape(schema).safeParse(obj);
+  return { result: toZodShape(schema).safeParse(obj), help: obj.flags.help as boolean ?? false };
```

Comment:
Unfortunately, If parse error occurs in Zod, we cannot access `data,` So the help option returns like this 😞 
https://github.com/colinhacks/zod#safeparse